plugins/LanguagesManager/LanguagesManager.php

```diff
@@ -138,6 +138,19 @@ public function uninstall()
     }
 
     /**
+     * @return boolean
+     */
+    public static function uses12HourClockForCurrentUser()
+    {
+        try {
+            $currentUser = Piwik::getCurrentUserLogin();
+            return API::getInstance()->uses12HourClockForUser($currentUser);
```

Comment:
Ideally, we would use something like `Request::processRequest()` here. Otherwise, if a plugin listens to this API, the result might be different when calling it directly see https://github.com/piwik/piwik/issues/7105